tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -3768,11 +3768,24 @@ void MSMain(uint GID : SV_GroupIndex,
       <RootValue Index="2" ResName="InputVector2" />
       <RootValue Index="3" ResName="OutputVector" />
     </RootValues>
-    <!-- This shader requires the following defines to be passed in as arguments:
-     // 1 and 2 are required to compile the shader.
-     // 1. -DOPERATOR : "*" "+" "," "-" "/" etc.
-     // 2. -DOPERAND2: : InputVector2, ScalarInput, or "" depending on the test.
-     // Other defines are optional and are used to test different functions.
+    <!-- This shader has the following defines to be passed in as arguments:
+     TYPE : The type of the input vector, e.g. float, double, int, uint.
+     OUT_TYPE : The type of the output vector, e.g. float, double, int, uint.
+                In most cases will OUT_TYPE == TYPE.
```

Comment:
Is this a question or a statement? The `will` is throwing me off

Reply:
lol. I was typing out a comment and co-pilot made a suggestion that I autocompleted.  
Supposed to just be 'In most cases OUT_TYPE == TYPE'.